src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -156,23 +144,21 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         }
     };
 
-    private togglePanAndZoomEvents = (): void => {
+    private toggleInteractionEvents = (): void => {
         const isViewportFreezed = this.isViewportFreezed();
         if (!this.chart) {
             return;
         }
-        if (isViewportFreezed) {
-            this.chart[DRAG_PAN_EVENT].disable();
-            this.chart[SCROLL_ZOOM_EVENT].disable();
-        } else {
-            this.chart[DRAG_PAN_EVENT].enable();
-            this.chart[SCROLL_ZOOM_EVENT].enable();
-        }
+
+        INTERACTION_EVENTS.forEach(
+            (interactionEvent: string): void =>
+                this.chart[interactionEvent][isViewportFreezed ? "disable" : "enable"](),
+        );
```

Comment:
I think it could be to save perf a bit :)
```
        const action = isViewportFreezed ? "disable" : "enable";
        INTERACTION_EVENTS.forEach(
            (interactionEvent: string): void =>
                this.chart[interactionEvent][action](),
        );
```

Reply:
updated